eogrow/core/storage.py

```diff
@@ -40,7 +40,8 @@ class Schema(ManagerSchema, BaseSettings):
             )
         )
         structure: Dict[str, str] = Field(
-            default_factory=dict, description="A flat key: value store mapping each key to a path in the project."
+            default_factory=Dict[str, str],
```

Comment:
Interesting, I though these defaults had to be objects, not types. Does this actually initialize as an empty dictionary, if nothing is provided in some cases like below, where the default is an empty list?

Reply:
it actually fails, I was hoping it would work. Apparently i'll need to do something else.